libGitWrap/Commit.cpp

```diff
@@ -347,23 +345,23 @@ namespace Git
 
     DiffList Commit::diffFromParent(Result& result, unsigned int index)
```

Comment:
Could you please add documentation to this function? With `index` being totally ignored for a commit that has no parents, this isn't very obvious anymore...